### ISSUE TITLE
Can not create team on fresh Docker container

### ISSUE THREAD

**Issue by MaxWinklh2908** [deck9/input#164]
On self-hosted instances via Docker, the creation of a new team workspace is not possible any more. This blocks using the program completely. In version 1.8.4 it was still working.

## Steps to reproduce
- Starting with a fresh instance
- Executing:
```
VERSION_TAG="main"
docker volume create input-data
docker run -d -p 8080:8080 --name input -v input-data:/var/www/html/storage ghcr.io/deck9/input:$VERSION_TAG
```
- Opening `http://localhost:8080` in a browser

## Expected behavior (as with VERSION_TAG="v1.8.4")
- Get redirect to `http://localhost:8080/register`
- Entering some credentials and hitting `Register` will open `http://localhost:8080/teams/create`
- Entering a team name will open the workspace

## Actual behavior
- Get redirect to `http://localhost:8080/login`
- Need to hit `Create a new account` to be redirected `http://localhost:8080/register`
- Entering some credentials and hitting submit will open `http://localhost:8080/team-missing`
- Page displays: 
    - Join a Team to Get Started 
    - To begin using Input, you need to be part of a team. Currently, you're not in a team, which is a necessary step to access Input's capabilities.
    - If you've been invited to join a team, please check your email for the invitation link. 
If you haven't received an invite yet, reach out to your team administrator or the person managing your self-hosted instance to request access. 

I don't know how to keep going from here. As mentioned, it was working perfecly in 1.8.4

**Comment by PhilReinking** [comment]
@MaxWinklh2908 It appears that you already have a team in your database. The self-hosted version restricts you to creating only one team. The first registered user automatically becomes the admin and team owner. All subsequent users must be invited to this team through the admin panel.

I apologize that this information is not currently documented. This explains the message you encounter when attempting to register a second user.

I'm puzzled by your mention of a "fresh install," as I would have expected an empty database in that case, which would naturally result in different behavior.

**Comment by MaxWinklh2908** [comment]
Thank you for the answer. I digged a bit deeper and looked into the database. The problem is that the "fresh" instance doesn't come with an empty database as we both would expect. There already exists a team "Your Team" by user "mail@philippreinking.de".
```
sqlite> SELECT * FROM teams;
1|1|Your Team|0|2024-02-15 12:47:26|2024-02-15 12:47:26
sqlite> SELECT * FROM users;
1||mail@philippreinking.de||$2y$10$LAoSasFNYfmmVR9vvIWSrOlipBT33pIMFsnOntktUbUvpGmTUOSaa||1||l2bRbRHEHyaAAODB3jCI9R0E97wcpqwj|||||||2024-02-15 12:47:26|2024-02-15 12:55:36||
```

**Comment by PhilReinking** [comment]
oh ok, that is totally my fault, because it looks like I checked in my sqlite database for testing... will remove it and then it should be resolved in the next release

**Comment by PhilReinking** [comment]
should be fixed in [v1.9.2](https://github.com/deck9/input/releases/tag/v1.9.2)
